tasks/dummy-dockerfile/dummy-dockerfile.yaml

```diff
@@ -6,24 +6,26 @@ metadata:
 spec:
   description: >-
     Tekton task to create a new dockerfile and inject it into a workspace
-  params: 
+  params:
     - default: "source"
-      description: Subdirectory inside the `output` Workspace to clone the repo into.
+      description: directory in the `output` Workspace to clone the repo into.
       name: subdirectory
       type: string
-  workspaces: 
+  workspaces:
     - name: dockerfilews
       description: The workspace where the dockerfile is stored
       mountPath: /tmp
   steps:
-    - name: create-dockerfile  
-      image: registry.access.redhat.com/ubi9/ubi-minimal:9.3-1361.1699548032
+    - name: create-dockerfile
+      image: |
+        registry.access.redhat.com/ubi9/ubi-minimal:9.3-1361.1699548032
```

Comment:
You are adding a `\n` here that is making the pipeline fail

Reply:
i'm not sure how to change that, if i'm using it as a single-liner than it exceeds 80 chars and the yamllint fails

Reply:
Try putting it in between quotes